terraform/aks/config/review.tfvars.json

```diff
@@ -7,8 +7,6 @@
   "deploy_azure_backing_services": false,
   "enable_postgres_ssl": false,
   "key_vault_name": "s189t01-ccbl-rv-kv",
-  "key_vault_app_secret_name": "CCBL-APP-SECRETS-REVIEW",
-  "key_vault_infra_secret_name": "CCBL-INFRA-SECRETS-REVIEW",
   "enable_monitoring": false,
```

Comment:
this affects the workflows, so they will need to be updated before we can deploy the review app